config/locales/en.yml

```diff
@@ -139,4 +139,21 @@ en:
         message_box_info: 'You are viewing the relevant evidence summaries for thematic review: %{link}'
         evidence_hub_direct: 'Not what you were looking for? Go to %{link} for more information'
     news:
-      more_individual: "Read more" 
+      more_individual: "Read more"
+    get_involved:
+      list:
+        - title: Sign up to the IMPACT Principles
+          content_html: |
+            Evidence and evaluation is key to the aims of the Financial Capability Strategy – we want to help organisations generate more evidence around which financial capability interventions are most effective. The IMPACT principles set out the approach to evidence and evaluation the Strategy wants to embed in all organisations it works with. Signatories to the <a href="impact-principles">IMPACT principles</a> will act as “impact champions” helping to embed best practice within their organisations.
+        - title: Use our Evidence Hub
+          content_html: |
+           The <a href="evaluation-toolkit-homepage">Evaluation Toolkit</a> contains guidance and practical tools (including a set of Outcomes Frameworks) that you can use to plan and carry out your evaluation. The <a href="evidence_hub">Evidence Hub</a> is a focal point for evidence on the effectiveness of financial capability interventions. Any evaluation activity carried out with the <a href="evaluation-toolkit-homepage">Evaluation Toolkit</a> can be uploaded and shared to the <a href="evidence_hub">Evidence Hub</a>. That allows you to share your evaluation findings and best practice with the sector.
```

Comment:
As per previous question about urls.
Just want to flag this to @margOnline

Reply:
The best ways are to use interpolation passing the link as parameter (using the _path and the command described above).

For example on the yaml:
```yaml
content_html: |
  This is a text, with a %{href} inside.
```

And then on the view you can use passing the href as parameter:

```
<%= t("content_html", href: link_to('Some link', some_link_path)) %>
```

Reply:
@tomas-stefano @margOnline I've added the links to this page as suggested (last commit). As part fo that I took out the .each in be_involved.html.erb because it became overcomplicated with many different links in those pieces of text. Could one of you have a look to see if it looks OK as is and if there is a more elegant way to implement the list? Cheers.

Reply:
My instinct tells me there is a more elegant way but I think what you've done is good. Possible refactoring may become evident with additional static pages 👍